Readme.md

```diff
@@ -29,12 +29,10 @@ dnf install perl-Image-ExifTool
 
 ### Clone the Elodie repository
 
-You can clone Elodie from GitHub. You'll need `git` installed ([instructions](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)).
+You can clone Elodie from GitHub. You'll need `pip` and `git` installed ([instructions](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)).
```

Comment:
Can update this to either clone from Github or install via pip. Don't need both.

Reply:
I believe you do still need git installed if you're installing via pip+git. If Elodie is accepted into the pypa public python repository then you wouldn't need git anymore.